src/oas/__tests__/__snapshots__/operation.test.ts.snap

```diff
@@ -0,0 +1,586 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`oas operation openapi v2 1`] = `
```

Comment:
I opted for a operation snapshot test because it's easy and efficient to verify resolve results.

Reply:
On the other hand, I see this object is just gigantic and there's no way somebody is going to review this line by line. I suggest to make few and focused assertions rather than drop this here.

Reply:
🤷‍♂️ I went through it line by line and found it quite useful to verify all the right things are happening.